core/src/main/java/io/neow3j/script/VerificationScript.java

```diff
@@ -125,7 +125,11 @@ public int getSigningThreshold() {
             return 1;
         } else if (isMultiSigScript()) {
             try {
```

Comment:
This looks a bit complicated. I think we can merge the two try clauses into a single one. I had to create two separate suggestions since I can't suggest over deleted lines here in GitHub reviews.

> fyi, if you accept, you can batch the suggestions together in the `Files changed` view to create just a single commit with all suggestions.

1/2

```suggestion
            try (BinaryReader reader = new BinaryReader(script)) {
```